doc/development/deprecations.rst

```diff
@@ -54,6 +54,25 @@ Pending deprecations
   - Deprecated in v0.32
   - Will be removed in v0.33
 
+* The method ``tape.unwrap()`` and corresponding ``UnwrapTape`` class are deprecated, and
```

Comment:
Do we also need to explicitly deprecate `qml.tape.Unwrap`? Forgot they were different classes. @trbromley

Reply:
nope.  it is a slight variation on `tape.unwrap` and `UnwrapTape`.

Reply:
Added deprecation for `Unwrap` :)